xline/src/server/command.rs

```diff
@@ -208,6 +430,44 @@ where
     }
 }
 
+/// Generate `Command` proposal from `Request`
+pub(super) fn command_from_request_wrapper<S>(
```

Comment:
The command_from_request_wrapper was moved to xlineapi::command.

Reply:
Fixed